### DISCUSSION TITLE
2010/12/26/%E4%BA%BA%E5%8F%AF%E4%BB%A5%E8%A7%82%E6%B5%8B%E5%88%B0%E6%9C%AA%E6%9D%A5%E5%90%97

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#310] [Announcements]
# 2010/12/26/%E4%BA%BA%E5%8F%AF%E4%BB%A5%E8%A7%82%E6%B5%8B%E5%88%B0%E6%9C%AA%E6%9D%A5%E5%90%97

科幻小说里经常有回到过去或看见未来之类的情节。我今早上想一件事情的时候，突然有种感觉：以前似乎预见过这件事情的发生，也许在梦中梦见过吧。于是我就想，人类是不是真有这种可能：观测到未来的情景？我觉得这在逻辑上是行不通的。

https://blog.qizhen.xyz/2010/12/26/%E4%BA%BA%E5%8F%AF%E4%BB%A5%E8%A7%82%E6%B5%8B%E5%88%B0%E6%9C%AA%E6%9D%A5%E5%90%97

**Comment by ruanqizhen** [comment]
之前博客下的留言：

wh0m
2010年12月26日 22:25 编辑
你这是Dejavu吧。

回复
